crates/provider-sqldb-postgres/wit/deps.toml

```diff
@@ -0,0 +1 @@
+postgres = "https://github.com/vados-cosmonic/wit-wasmcloud-postgres/archive/main.tar.gz"
```

Comment:
I think we'd want this interface in the wasmCloud repo right?

Reply:
Definitely! Once it's stabilized I'll be happy to move it over -- I'm thinking we can move it to `wasmcloud/wit-wasmcloud-postgres`?

Reply:
I think matching it with the other wasi proposals would just be `wasmcloud/wasmcloud-postgres` right? Let me know if you need repos created/to accept a transfer

Reply:
Sure -- I personally like the prefixing of repos with what they contain but `wasmcloud/wasmcloud-postgres` works as well!

Reply:
Done! we're now using the following artifact:

https://github.com/wasmCloud/wasmCloud/releases/download/wit-wasmcloud-postgres-v0.1.0-draft/wit-wasmcloud-postgres-0.1.0-draft.tar.gz